scripts/abi_artifact_generator/templates/functions.inc.jinja_template

```diff
@@ -40,61 +48,24 @@ void __attribute__((noinline, weak)) ABIDEF /*( name )*/(
 /* Helps avoiding compiler warnings */
 void *typeless_setup = (void *) set_return_value_up;
 
-#ifdef __SIZEOF_INT128__
-#define INT128_IS_SCALAR 1
-#else
-#define INT128_IS_SCALAR 0
+#ifdef _MSC_VER
+#pragma warning(disable: 4715)
```

Comment:
What is this? Docs

Reply:
Closing all the doc-related threads for now.